tests/testRepository.py

```diff
@@ -64,11 +56,11 @@ def createDataRepository(mapperName, inputPath):
     return repoPath
 
 
-class IngestRawTestCase(utilsTests.TestCase):
+class IngestRawTestCase(lsst.utils.tests.TestCase):
     """Ensure that we can ingest raw data to a repository."""
-    @requirePackage(TEST_DATA_PACKAGE)
+    @unittest.skipIf(testDataPackage is None, "%s not setup"%(testData))
```

Comment:
I'd prefer to say "is not available" here so as not to imply a specific tool. Also, spacing around the `%`.